templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="/">Manufacturers (Coming Soon)</a>

```

Comment:
This link is not yet implemented. Consider commenting it out or removing it until the feature is ready.